.travis.yml

```diff
@@ -75,6 +70,6 @@ before_script:
 
 script:
   # Run the PHPUnit tests which also include the kernel tests.
-  - ./vendor/bin/phpunit -c ./core/phpunit.xml.dist ./modules/rules
+  - ./vendor/bin/phpunit -c ./core/phpunit.xml.dist ./modules/$MODULE
   # Check for coding standards.
-  - cd modules/rules && ./vendor/bin/phpcs --colors --report-width=130
+  - cd modules/$MODULE && ./vendor/bin/phpcs --colors --report-width=130
```

Comment:
This `cd modules/` is not required, is it? The current directory is already at Rules as a result of `before_script`

Reply:
Timing! I had this very minute discovered what the problem was, and about to post it here. I tried exactly the same correctiong, but in my Travis build I get $module empty, it reports
`Cannot open file "./modules//tests.php"`